### ISSUE TITLE
How to update a file item in chat room? 

### ISSUE THREAD

**Issue by SAEEDHONARI** [vector-im/element-android#3109]
I need to update a file item again after downloading, please help

**Comment by SimonBrandner** [comment]
I don't think there is any way to update a file you've already sent at the moment (at least not using UI)

**Comment by bmarty** [comment]
Yes, it's not possible so far. You can redact (= delete) the message and upload a new file.
